understanding/20/audio-description-or-media-alternative-prerecorded.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Description or Media Alternative (Prerecorded)</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Prerecorded videos can be understood by more people</dd>
-		<dt>Author task</dt><dd>Provide a description of the visual content in videos</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Prerecorded videos can be understood by more people.</dd>
+         <dt>What to do</dt><dd>Provide a description of the visual content in videos.</dd>
+         <dt>Why it's important</dt><dd>People who are blind or have low vision can understand multimedia content.</dd> 
```

Comment:
```suggestion
         <dt>Why it's important</dt><dd>People who can’t fully see can still understand multimedia.</dd> 
```

Reply:
hummm -- I think "People who are blind or have low vision" is stronger. It's not "multimedia", it's video. Also I think good to be more specific that "understand it". /me looks through https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/ ... those are much simpler.

Perhaps just
> What to do: Provide a description of the visual content in videos.
> Why it's important: People who are blind and some with low vision content can't see the visual content.

Reply:
```suggestion
         <dt>Why it's important</dt><dd>People who are blind and some with low vision can't see the visual content.</dd> 
```